rocksdb_admin/rocksdb_admin.thrift

```diff
@@ -148,6 +160,13 @@ service Admin {
 void ping()
 
 /*
+ * Add the DB to the host, throw exception if it already exists
+ * and overwrite is false.
```

Comment:
I thought the idea is that:
1) if the process already manages the db, we should always throw an exception.
2) otherwise, if overwrite is true, we will remove the DB content on the host, which might be leftover previously.